pkgs/json_syntax_generator/lib/src/include/helper_library.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Helper methods for the code generator that are added to the generated file.
```

Comment:
Does the path `include/...` still makes sense now that it's going to be copied everywhere?